lang/pypy/Portfile

```diff
@@ -54,8 +71,12 @@ array set module_scripts {
     smtpd.py smtpd
 }
 
-subport pypy36 {
-    set python.branch 3.6
+worksrcdir          pypy-${github.tag_prefix}${version}
+
+subport pypy38 {
+    set python.branch 3.8
+    version $pypy_versions(3.8)
+    github.tag_prefix release-pypy3.8-v
```

Comment:
These two lines are repeated in each subport, varying only by the python version. Can you deduplicate this and write those lines only once, making use of the `python.branch` variable you defined?